README.md

```diff
@@ -2,10 +2,10 @@
 
 An engineer working for a [CSE](./CSE.md) project...
 
-* Has responsibilities to their team – mentor, coach, and lead​
+* Has responsibilities to their team – mentor, coach, and lead​.
 * Knows their **playbook**. Follows their playbook.  Fixes their playbook if it is broken.  If they find a better playbook, they copy it. If somebody could use your playbook, give them yours.​
-* Leads by example.  Models the behaviors we desire both interpersonally and technically. (this is what most people think of as a “developer”)​
-* Strives to understand how their work fits into a broader context and ensures the outcome
+* Leads by example.  Models the behaviors we desire both interpersonally and technically.(This is what most people think of as a “developer”)​
```

Comment:
Can you delete the period at the end of the sentence and replace it with a space instead of changing the capitalization?